File: 01~10/08/remove_duplicate.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        
        nw_index = 0
        for i in range(len(nums)):
            if nums[nw_index] != nums[i]:
                nw_index +=1
                nums[nw_index] = nums[i]
        return (nw_index + 1)

File: 01~10/08/remove_duplicate.go
func removeDuplicates(nums []int) int {
	nw_index := 0
	for _, num := range nums {
		if nums[nw_index] != num {
			nw_index++
			nums[nw_index] = num
		}
	}

	return (nw_index + 1)
}